### PR TITLE
add section on creating new QN video categories to operator guide

### PR DESCRIPTION
**Pull request by kdembler**
I've noticed there isn't any resource describing how to manage metaprotocol categories, so I added a small section to the operator guide for future reference

### CODE REVIEW

File: docs/operator-guide.md [hunk 1]
```diff
@@ -115,6 +115,27 @@ Atlas uses list of categories defined in `content.categories` config entry to de
 
 Atlas will, by default, only display content belonging to one of the defined display categories in the app. That means that video with a category that doesn't belong to any local category will not be available in the app. If you want to change that behavior, you can set `content.showAllContent` to `true` in the config file. That will make the app display all the content, regardless of the category.
 
+You are free to use existing metaprotocol (Query Node) categories, buy you can also create your own that match your Gateway's specific needs. To get a list of all the existing categories you can use the following QN query:
```
Comment (attemka) [inline docs/operator-guide.md]: buy you can -> but you can

Comment (drillprop) [inline docs/operator-guide.md]:
```suggestion
You are free to use existing metaprotocol (Query Node) categories, but you can also create your own that match your Gateway's specific needs. To get a list of all the existing categories you can use the following QN query:
```

**Review by attemka — APPROVED** [review]
lgtm aside of minor typo

**Review by drillprop — APPROVED** [review]
Just a typo, LGTM

**Comment by kdembler** [comment]
Thanks guys 🫡

**Pull request merged**